googleMapsApiModule.py

```diff
@@ -73,11 +73,22 @@ def mapsstatic(search_location):
         result_value(int): elevation(in metres) above/below sea level
     """
 
-
+@logger_config.logger
 def elevation(search_location):
     result = gmaps.geocode(search_location)
     json = requests.get(f'https://maps.googleapis.com/maps/api/elevation/json?locations={result[0]["geometry"]["location"]["lat"]},{result[0]["geometry"]["location"]["lng"]}&key={config.key}').json()
     result_value = json['results'][0]['elevation']
     position = "above" if result_value > 0 else "below"
     print(f'{search_location} is {round(result_value,2)} metres {position} sea level')
-    return result_value
\ No newline at end of file
+    return result_value
+
+@logger_config.logger
+def places(search_location):
+    address = search_location
+    json = requests.get(f'https://maps.googleapis.com/maps/api/place/findplacefromtext/json?input={address.lower().replace(" ", "+")}&inputtype=textquery&fields=photos,formatted_address,place_id&key={config.key}').json()
```

Comment:
@vishakha-lall  I'd really like to see all these URLs in the constants file. I'm not quite sure why we've restricted that to the direction and geocoding links specifically. Thoughts?

Reply:
Certainly, you are right @janakrajchadha. Please make the changes @adityabisoi

Reply:
@janakrajchadha @vishakha-lall I have updated the changes.